drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -145,6 +145,11 @@ def self.static_options_for_type(pick_list_type, user:)
         end
 
         form_types.map { |ft| { code: ft[:value], label: ft[:desc] } }
+      when 'CONTINUUM_PROJECTS'
+        Hmis::Hud::Project.where(continuum_project: true).
```

Comment:
This project list should be limited to projects in the current HMIS data source. Typically we rely on `viewable_by(user)` for this. In this case it sounds like we don't want the limit the list to viewable projects, but it should still be limited to continuum projects _in the HMIS_.

Reply:
Thanks for the catch, @gigxz